WalletWasabi/Wallets/Wallet.cs

```diff
@@ -575,6 +581,8 @@ public void ExcludeCoinFromCoinJoin(OutPoint outpoint, bool exclude = true)
 		UpdateExcludedCoinFromCoinJoin();
 	}
 
+	public void SetFilePath(string path) => KeyManager.SetFilePath(path);
```

Comment:
Method seems redundant IMO.

If you're in a place where you have access to `Wallet` (such as `WalletModel`) then you also have access to `Wallet.KeyManager`. Might as well just use it directly in the calling code.

Reply:
Done!